client/app/(auth)/_layout.js

```diff
@@ -0,0 +1,10 @@
+import { AuthProvider } from '../../context/auth';
```

Comment:
Just curious, what purpose does having (auth) as well as (app) do? If possible, i think we may want to go to (app) only if not needed.